src/Aspire.Dashboard/Components/Pages/Resources.razor

```diff
@@ -70,66 +71,62 @@
                                 SelectedValue="@SelectedResource"
                                 ViewKey="ResourcesList">
                 <Summary>
-                    @{
-                        var gridTemplateColumns = HasResourcesWithCommands ? "1fr 1.5fr 1.25fr 1.5fr 2.5fr 2.5fr 1fr 1fr 1fr" : "1fr 1.5fr 1.25fr 1.5fr 2.5fr 2.5fr 1fr 1fr";
-                    }
                     <FluentDataGrid Virtualize="true"
                                     GenerateHeader="GenerateHeaderOption.Sticky"
                                     ItemSize="46"
                                     Items="@FilteredResources"
                                     ResizableColumns="true"
-                                    GridTemplateColumns="@gridTemplateColumns"
+                                    GridTemplateColumns="@GetGridTemplateColumns()"
                                     RowClass="GetRowClass"
                                     Loading="_isLoading"
-                                    ShowHover="true"
-                                    TGridItem="ResourceViewModel"
-                                    OnRowClick="@(r => r.ExecuteOnDefault(d => ShowResourceDetailsAsync(d, buttonId: null)))"
-                                    Class="enable-row-click">
```

Comment:
Why are you removing this?

Reply:
I don't know what happened. Re-added to all 3